crates/core/src/node/in_memory.rs

```diff
@@ -77,7 +77,9 @@ use zksync_types::{
 };
 
 /// Max possible size of an ABI encoded tx (in bytes).
-pub const MAX_TX_SIZE: usize = 1_000_000;
+/// NOTE: this deviates slightly from the default value in the main node config,
```

Comment:
Assuming this value will also be bumped in the main node once the related contract changes are introduced?